guide/source/prepare-meteor-3.0.md

```diff
@@ -116,6 +116,80 @@ Note that we're not using the if (loading) anymore. To see a practical project,
 If you use `Tracker.autorun()`, for example, reading about the tracker with the [async callback function](https://blog.meteor.com/new-meteor-js-2-10-and-the-async-tracker-feature-ffdbe817c801) is also recommended.
 
 
+## Changes for packages
+
+### Meteor.isFibersDisabled
+
+You can use the [`Meteor.isFibersDisabled`](https://github.com/meteor/meteor/blob/6ac474627a4d2536090484eb95e7c021370aaefe/packages/meteor/asl-helpers-client.js#L1-L8) property to check if the current Meteor version
+is using Fibers or not. In all releases before Meteor 3.0 this property will be `falsy`(`undefined`).
+In Meteor 3.0 this property will be return `true`.
+
+Wich means that you can have a code like this:
+
+```js
+
+if (Meteor.isFibersDisabled) {
+  // Meteor 3.0
+} else {
+  // Meteor 2.x
+}
+
+```
+
+### Changes for packages that are client only
+
+If your package is client only, you don't need to worry about the async changes. You can just update your package to be compatible with Meteor 3.0 by adding the following line to your `package.js`:
+
+```js
+Package.onUse((api) => {
+  api.versionsFrom(['1.10', '2.3', '3.0-alpha.19']);
+  //                            ^^^^^^^ for testing your package with meteor 3.0
+
+  api.versionsFrom(['1.10', '2.3', '3.0']);
+  //                           ^^^^^^^ for meteor 3.0
+});
+```
+
+If you want to an example of this change you can take a look in this [commit](https://github.com/meteor/react-packages/commit/96313a1afcc41ef9a23c7496470b375e7d357793)
+where we made possible to a package be used in Meteor 3.0.
+
+This change makes sure that your package is still compatible with Meteor 2.x
+and also with Meteor 3.0.
+
+
+### Changes for packages that are do not using Meteor packages that had breaking changes
```

Comment:
`Changes for packages that do not use Meteor packages that had breaking changes`